File: Python/webhook/subscribe_webhook.py
import requests

userID = "" #provide your userId here 
webhooks = [
        "https://example.com/webhook","https://example1.com/webhook"
        ]

url = "https://mediafirewall-ai.millionvisions.ai/notification/webhook"

body = {
    "userId" : userID,
    "webhooks" : webhooks
}

headers = {'Content-Type': 'application/json','Authorization':'Bearer token'}

params = {
  'apikey': '' #provide your apikey here
}

response = requests.post(url,params=params,json=body ,headers=headers)

if response.status_code == 201:
    print("Webhook request sent successfully. Please check the webhook message and click on the SubscribeUrl.")       
else:
    print("Failed to send POST request. Status code:", response.status_code)
    print("Response content:", response.content)

File: JavaScript/subscribe_webhook.js
const fetch = require('node-fetch');

const userID = ""; // Provide your userId here
const webhooks = [
    "https://example.com/webhook",
    "https://example1.com/webhook"
];
const url = "https://mediafirewall-ai.millionvisions.ai/notification/webhook";
const body = {
    userId: userID,
    webhooks: webhooks
};
const headers = {
    'Content-Type': 'application/json',
    'Authorization': 'Bearer token'
};
const params = new URLSearchParams({ apikey: '' }); // Provide your apikey here

async function sendPostRequest() {
    try {
        const response = await fetch(`${url}?${params.toString()}`, {
            method: 'POST',
            headers: headers,
            body: JSON.stringify(body)
        });
        if (response.ok) {
            console.log("webhooks request sent successfully.");
        } else {
            console.log("Failed to send webhook request. Status code:", response.status);
            const responseText = await response.text();
            console.log("Response content:", responseText);
        }
    } catch (error) {
        console.error("Error occurred:", error);
    }
}

sendPostRequest();
